build.gradle

```diff
@@ -2,73 +2,22 @@ import org.jetbrains.dokka.gradle.DokkaTask
-      // The benefit is that depending clients do not have to upgrade to Kotlin 1.4
-      'kotlinCompiler': '1.4.21',
-      'kotlinLib': '1.3.72',
-      // 2.7.0 to ensure support for setExpedited().
-      'workManager': '2.7.0',
-  ]
-  ext.deps = [
-      assertj_core: 'org.assertj:assertj-core:3.9.1',
-      // We don't need the latest version of AndroidX (there are no bugs that impact what LeakCanary
-      // relies on), we're sticking a bit older because most apps will be using a more recent version
-      // and they'll automatically resolve to higher version without having to necessarily resort to a
-      // resolution strategy.
-      androidGradlePlugin: "com.android.tools.build:gradle:4.2.2",
-      dokkaPlugin: "org.jetbrains.dokka:dokka-gradle-plugin:1.6.0",
-      mavenPublishPlugin: "com.vanniktech:gradle-maven-publish-plugin:0.18.0",
-      androidx: [
-          annotation: 'androidx.annotation:annotation:1.0.2',
-          core: 'androidx.core:core:1.0.1',
-          fragment: 'androidx.fragment:fragment:1.0.0',
-          test: [
-              core: 'androidx.test:core:1.0.0',
-              espresso: 'androidx.test.espresso:espresso-core:3.1.0',
-              rules: 'androidx.test:rules:1.1.0',
-              runner: 'androidx.test:runner:1.1.0',
-              orchestrator: 'androidx.test:orchestrator:1.1.0',
-          ],
-          work: [
-            // Note: we use the Java artifact because the Kotlin one bundles coroutines.
-            runtime: "androidx.work:work-runtime:${versions.workManager}",
-            multiprocess: "androidx.work:work-multiprocess:${versions.workManager}",
-          ]
-      ],
-      android_support: 'com.android.support:support-v4:28.0.0',
-      clikt: 'com.github.ajalt:clikt:2.3.0',
-      curtains: 'com.squareup.curtains:curtains:1.2.2',
-      jline: 'jline:jline:2.14.6',
-      detekt: 'io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.6.0',
-      junit: 'junit:junit:4.12',
-      kotlin: [
-          binaryCompatibilityValidatorPlugin: "org.jetbrains.kotlinx:binary-compatibility-validator:0.8.0",
-          gradlePlugin: "org.jetbrains.kotlin:kotlin-gradle-plugin:${versions.kotlinCompiler}",
-          stdlib: "org.jetbrains.kotlin:kotlin-stdlib:${versions.kotlinLib}",
-          reflect: "org.jetbrains.kotlin:kotlin-reflect:${versions.kotlinLib}"
-      ],
-      kotlin_statistics: 'org.nield:kotlin-statistics:1.2.1',
-      mockito: 'org.mockito:mockito-core:3.5.10',
-      mockito_kotlin: 'com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0',
-      okio: 'com.squareup.okio:okio:2.2.2',
-      okio_1x: 'com.squareup.okio:okio:1.14.0',
-      robolectric: 'org.robolectric:robolectric:4.0-alpha-3',
+    'minSdk'    : 14,
+    'compileSdk': 31,
   ]
   repositories {
     google()
     gradlePluginPortal()
-    // For binary compatibility validator.
-    maven { url "https://kotlin.bintray.com/kotlinx" }
+    mavenCentral()
```

Comment:
Did binary compatibility validator move to central? Feels like this should be a separate change.

Reply:
Yeah, see [mavenCentral](https://search.maven.org/artifact/org.jetbrains.kotlinx.binary-compatibility-validator/org.jetbrains.kotlinx.binary-compatibility-validator.gradle.plugin/0.8.0/pom).